doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-create-test-template-dmn-based-proc.adoc

```diff
@@ -0,0 +1,34 @@
+[id='test-designer-create-test-template-dmn-based-proc']
+= Creating a test scenario template for DMN-based test scenarios
+
+Create a test scenario template for DMN-based scenarios by following the procedure below to validate your rules and data.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and click the project that you want to create the test scenario for.
+. Click *Add Asset* -> *Test Scenario*.
+. Enter a *Test Scenario* name and select the appropriate *Package*. The package you select must contain all the required data objects and rule assets assigned.
+. Select *DMN* as the *Source type*.
+. Select an existing DMN asset using the *Choose DMN asset* option.
+. Click *Ok* to create and open the test scenario in the *Test Scenario* designer.
+. To map the *GIVEN* column header to a data object:
```

Comment:
Starting from this point the rest of the chapter is not correct. The scenario template of a DMN scenario is automatically generated during file creation based on DMN model: each input node becomes one or more GIVEN columns and each decision node becomes one or more EXPECT columns.
What you are describing here is correct but it is useful only for edit the template: user can decide to remove some columns and then add them again (for instance to move a decision as input)

Reply:
@danielezonca, I have made the changes as per Klara's feedback in the QE review. Refer to [Klara's QE review](https://issues.jboss.org/browse/BXMSDOC-3836) for more details.